Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -194,14 +194,14 @@ public extension CurrentChatUserController {
     ///   - role: The role for the user.
     ///   - teamsRole: The role for the user in a specific team. Example: `["teamId": "role"]`.
     ///   - userExtraData: Optionally provide new user extra data to be updated.
-    ///   - unset: Existing values for specified properties are removed. For example, `image` or `name`.
+    ///   - unsetProperties: Remove existing properties from the user. For example, `image` or `name`.
     ///   - completion: Called when user is successfuly updated, or with error.
     func updateUserData(
         name: String? = nil,
         imageURL: URL? = nil,
         privacySettings: UserPrivacySettings? = nil,
         role: UserRole? = nil,
-        teamsRole: [String: String]? = nil,
+        teamsRole: [TeamId: UserRole]? = nil,
```

Comment:
TeamId is not a breaking change. But UserRole it is kinda a breaking change. IMO it is not a big deal, but let me know WDYT. Overall it would be nice we keep the consistency of the types here.

Reply:
that's fine - we don't advertise this feature for now anyway.